pkg/actions/lua/databricks/client.go

```diff
@@ -8,6 +8,8 @@ import (
 	"regexp"
 	"strings"
 
+	luautil "github.com/treeverse/lakefs/pkg/actions/lua/util"
+
```

Comment:
```suggestion
	luautil "github.com/treeverse/lakefs/pkg/actions/lua/util"
```

Reply:
Don't mess with it, just remove the redundant newline 📏

Reply:
sure thing sure thing